docs/source/contribute.rst

```diff
@@ -9,44 +9,53 @@ Researchers
+Please first fill out this VERY brief survey about the consent forms
+for your studies: http://goo.gl/forms/2lsmYcOsAs . It only has a few
+questions and should take a few minutes to fill out.  Even if
+your consent form doesn't yet include any provision for data sharing,
+your contribution would be very valuable (although it would consist of
+simply saying "No").
+
+Additional Materials
 ^^^^^^^^^^^^^^^^^^^^
 
-Please report to `GitHub issues`_ or even send a new pull request via
-`GitHub pull requests`_ with
+We're always looking for new materials to add to this shared neuroimaging resource.  In particular, we're looking for:
 
-- samples of consent forms allowing re-distribution/deposit to
-  public archives
+- Samples of consent forms allowing re-distribution/deposit to
+  public archives 
 
-- relevant publications and discussions
+- Relevant publications and discussions
 
-- changes/recommendations for the **ultimate** consent form formulation
+- Changes/recommendations for the **ultimate** consent form formulation
+
+To add to the materials on this site, please open an issue on our `GitHub issues`_ page or send a new pull request via `GitHub pull requests`_.  Whether or not you should open an issue or make a pull request depends on the type of contribution you are making.
+
+For sample consent forms and links to relevant publications, please make submissions exclusively via the `GitHub issues`_ page.  When submitting consent forms, please include a full URL to the form and the desired filename you'd like to see it represented as on the samples_ page.  Note that for the URL you submit, `persistent URLs`_ such as `DOIs`_ are ideal, since these will not require frequent updates to this site in the event that a link moves.  If a consent form does not have a DOI associated with it, an `Internet Archive Wayback Machine`_ saved page also works well in practice.
+
+Changes to the ultimate consent form should be made via `GitHub pull requests`_.
 
 IRB committee members
 ---------------------
 
-We would welcome your feedback  very much, in particular
+We would welcome your feedback very much, in particular:
 
-- what concerns on public sharing of neuroimaging data you might have
-  if any identifiable information removed (e.g. skull stripped) and
-  research participants agreed to those terms?
+- What concerns on public sharing of neuroimaging data you might have
+  if all identifiable information is removed (e.g. skull stripped) and
+  research participants agreed to those terms.
 
-- what particular consent form composition and wording aspects would
-  you recommend? (e.g. "make it an explicit additional form requiring
+- What particular consent form composition and wording aspects would
+  you recommend (e.g. "make it an explicit additional form requiring
   a separate signature") and why?
 
 Contact information
 ===================
 
-- directly via email open-brain-consent@datalad.org
+- Directly via email at open-brain-consent@datalad.org
```

Comment:
Crap, nobody cared to merge my https://github.com/con/open-brain-consent/pull/57/files so it will conflict... But probably to merge it after this one now